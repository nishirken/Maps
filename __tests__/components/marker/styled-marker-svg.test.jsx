import StyledMarkerSvg from 'Components/marker/styled-marker-svg';
import 'jest-styled-components';

describe('Rendered', () => {
    const TestStyledMarkerSvg = shallow(<StyledMarkerSvg />);

    it('Rendered', () => {
        expect(TestStyledMarkerSvg).toMatchStyledComponentsSnapshot();
    });

    it('Should change transform style property, when getting a center property', () => {
        TestStyledMarkerSvg.setProps({ current: true });
        expect(TestStyledMarkerSvg).toMatchStyledComponentsSnapshot();
    });
});
