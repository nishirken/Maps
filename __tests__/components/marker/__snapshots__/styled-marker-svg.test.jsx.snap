// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendered Rendered 1`] = `
.bxjcOQ {
  width: 15px;
  height: 15px;
  fill: #E53935;
  cursor: pointer;
  transition: [object Object] 0.4s ease-in-out;
  -webkit-transition: [object Object] 0.4s ease-in-out;
  transform: scale(1);
  -ms-transform: scale(1);
  -webkit-transform: scale(1);
}

.bxjcOQ:hover {
  fill: #8C2321;
}

<svg
  className="bxjcOQ"
/>
`;

exports[`Rendered Should change transform style property, when getting a center property 1`] = `
.bxjcOQ {
  width: 15px;
  height: 15px;
  fill: #E53935;
  cursor: pointer;
  transition: [object Object] 0.4s ease-in-out;
  -webkit-transition: [object Object] 0.4s ease-in-out;
  transform: scale(1);
  -ms-transform: scale(1);
  -webkit-transform: scale(1);
}

.bxjcOQ:hover {
  fill: #8C2321;
}

<svg
  className="bxjcOQ"
/>
`;
